<launch>
	<arg name="namespace" default="starboard_cam"/>
	<arg name="object" default="totem"/>
	<arg name="color" default="red"/>
	<arg name="debug" default="true"/>

	<group ns="$(arg namespace)">
		<node
			pkg="robotx_vision"
			type="shape_roi"
			name="shape_roi"
			required="true"
			output="screen"
		>
		<param name="subscribed_image_topic" value="image_rect_color"/>
		<param name="object_shape" value="$(arg object)"/>
		<param name="object_color" value="$(arg color)"/>
		<param name="debug" value="$(arg debug)"/>
		<param name="published_topic" value="$(arg object)/$(arg color)/roi"/>

		</node>
	</group>
</launch>
